packages/experimental/Spinner/src/Spinner.styling.tsx

```diff
@@ -0,0 +1,49 @@
+import type { UseStylingOptions } from '@fluentui-react-native/framework';
+import { buildProps } from '@fluentui-react-native/framework';
+import type { SpinnerProps, SpinnerSlotProps, SpinnerTokens } from './Spinner.types';
+import { spinnerName } from './Spinner.types';
+import { defaultSpinnerTokens } from './SpinnerTokens';
+
+export const diameterSizeMap: { [key: string]: number } = {
```

Comment:
Does this need to be an exported const?

Reply:
hmm yes , we don't modify this anytime.

Reply:
I put it in an unclear way, I meant to check on the need for `exporting` this outside ( not the `const`) - if it's unused elsewhere outside this file, it can be const `diameterSizeMap`

Reply:
never mind, saw the reference in rendering code.